tfx/proto/example_gen.proto

```diff
@@ -71,10 +76,10 @@ message Input {
   //     latest version for the latest span, and store the version number in
   //     metadata.
   //   - If VERSION is specified, but not SPAN, an error will be thrown.
+  //   - It could also be optionally specified with the width in the same way
```

Comment:
under line 73, +
"//     e.g. 'data_{SPAN}/v{VERSION}/*' matches to files under 'data_0/v1/'
 //       to consume as input for Version-1 of Span-0."